rules-threat-hunting/windows/powershell/powershell_script/posh_ps_mailbox_access.yml

```diff
@@ -0,0 +1,23 @@
+title: Windows Mail App Mailbox Access Via PowerShell Script
+id: 4e485d01-e18a-43f6-a46b-ef20496fa9d3
+status: experimental
+description: Detects powershell script that try to access the Windows Mail app MailBox via code. This could be used by an attacker to exfiltrate or delete the content of the emails for example.
```

Comment:
```suggestion
description: Detects PowerShell scripts that try to access the default Windows MailApp MailBox. This indicates manipulation of or access to the stored emails of a user. E.g. this could be used by an attacker to exfiltrate or delete the content of the emails.
```

Reply:
merged the two descriptions :see_no_evil: